index.html

```diff
@@ -3632,7 +3632,7 @@ <h2>The <dfn>ThingDiscovery</dfn> interface</h2>
       The <dfn>active</dfn> property is `true` when the discovery is actively ongoing on protocol level (i.e. new <a>TD</a>s may still arrive) and `false` otherwise.
     </p>
     <p>
-      The <dfn>done</dfn> property is `true` if the discovery has been completed with no more results to report and <a>discovery results</a> is also empty.
+      The <dfn>done</dfn> property is `true` if the discovery has been completed with no more results to report.
```

Comment:
A good start, since we don't (necessarily) need that internal slot with asyncIterator - or we need to specify how does it work (for instance, `next()` could pop from that internal slot.